common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -195,6 +195,8 @@ public class BrokerConfig {
      * Whether to distinguish log paths when multiple brokers are deployed on the same machine
      */
     private boolean isolateLogEnable = false;
+    @ImportantField
+    private String zoneName;
```

Comment:
Currently, use the rpchook instead.

We need more time to prove this PR's general value.